planning/external_velocity_limit_selector/src/external_velocity_limit_selector_node.cpp

```diff
@@ -122,6 +143,8 @@ void ExternalVelocityLimitSelectorNode::onVelocityLimitFromInternal(
 
   const auto velocity_limit = getCurrentVelocityLimit();
   publishVelocityLimit(velocity_limit);
+
+  publishDebugString();
```

Comment:
It looks `velocity_limit` variable and debug string have the same data and they are published at the same time.
What kind of usage do you expect about the debug data? :thinking:

Reply:
The debug string outputs **velocity limit table** in the module. It shows different type data from velocity limit.
I intend to use the debug topic to check **velocity limit table** when there is an update of that.

Reply:
I see. `velocity_limit` only has the `hardest_limit` in the `velocity_limit_table`, and `velocity_limit_table` has all of the limits.
Thanks :+1: